assets/js/src/form_editor/store/map_form_data_after_loading.jsx

```diff
@@ -25,6 +25,10 @@ export default function mapFormDataAfterLoading(data) {
       placeSlideInFormOnAllPosts: data.settings.place_slide_in_form_on_all_posts === '1',
       slideInFormDelay: asNum(data.settings.slide_in_form_delay),
       slideInFormPosition: data.settings.slide_in_form_position,
+      borderRadius: asNum(data.settings.borderRadius),
```

Comment:
Maybe I missed that in the first review. Do we save those properties on API as underscore or camelcase? I think it should be as underscore so that it is consistent with other properties.